app/pulp/app/models/progress.py

```diff
@@ -58,6 +62,9 @@ class ProgressReport(Model):
     task = models.ForeignKey("Task", on_delete=models.CASCADE)
     suffix = models.TextField(default='')
 
+    _using_context_manager = False
+    _last_increment_time = None
```

Comment:
Since this is only set during save I think s/_last_increment_time/_last_save_time/